### PR TITLE
Model tests and factories

### PR DESCRIPTION
**Pull request by nunomsantos**
Ironically enough, one of these new tests is still [failing](https://travis-ci.org/engwebUM/parkyourcar/builds/62840800)  – but it should pass after rebasing from PR #29.

Further discussion needed on the necessity of the three tests being temporarily skipped.

### CODE REVIEW

File: spec/factories/attachments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'faker'
```
Comment (fredCarvalho) [inline spec/factories/attachments.rb]: Maybe you don't need this gem on this file

File: spec/factories/bookings.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require 'faker'
```
Comment (fredCarvalho) [inline spec/factories/bookings.rb]: And here faker is not necessary

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      end
+
+      it 'phone numbers' do
+        FactoryGirl.create(:user, phone_number: 999_999_999)
+        expect(FactoryGirl.build(:user, phone_number: 999_999_999)).not_to be_valid
+      end
+    end
+  end
+
+  describe '#photo' do
```
Comment (fredCarvalho) [inline spec/models/user_spec.rb]: Don't forget to implement these tests

**Comment by fredCarvalho** [comment]
After the corrections and rebasing to pass the tests you can merge.

**Comment by nunomsantos** [comment]
Thanks, @fredCarvalho, I've addressed most of your comments.

There are still some tests being temporarily skipped because they need further discussion. I think this discussion can be done later and that we should merge this branch as it is right now.

**Comment by fredCarvalho** [comment]
Ok. The tests you've done are very useful, so you can merge now :+1:

**Pull request merged**